test/unit/logic/block.js

```diff
@@ -562,9 +562,78 @@ describe('block', () => {
 			}).to.throw();
 		});
 
-		it('should return verification response for a given block', () => {
+		it('should return positive for a good block', () => {
 			return expect(block.verifySignature(blockData)).to.be.true;
 		});
+
+		it('should return negative for a block with modified version', () => {
```

Comment:
Please change those descriptions, as `negative` is more like negative number. For example: `should fail for a block...` or `should return false for...`